integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1153,6 +1161,244 @@ private static String getForwardedPort(String portForwardFileName) {
+    final String wlstScriptFilePath = RESOURCE_DIR + "/python-scripts/" + wlstScriptFileName;
+    final String wlstScriptDestPath = "/tmp/" + wlstScriptFileName;
+    final String wlstPropDestPath = "/tmp/connect.prop";
+    final String diiImageName = "wls-docker-container-image";
+    final String diiModelFileName = "dii-docker-container.yaml";
+    final String diiModelPropFileName = "dii-docker-container.properties";
+    ExecResult result = null;
+
+    try {
+      // create a dii images to create a WebLogic docker container
+      String diiDomainImage = createDiiImageAndVerify(domainUid, domainNamespace,
+          diiImageName, diiModelFileName, diiModelPropFileName, null);
+      logger.info("Created dii image: {0}", diiDomainImage);
+
+      // create a WLS docker container using the dii images created above
+      result = createAndStartWlsDockerContainerAndVerify(domainUid, containerName, diiDomainImage);
+      if (result.exitValue() == 0) {
+        logger.info("Create WLS docker container succeeded: {0}", result.stdout());
+      } else {
+        logger.info("Create WLS docker container failed: {0}", result.stderr());
+      }
+
+      // create WLST property file
+      File wlstPropertiesFile = assertDoesNotThrow(() -> File.createTempFile("wlst", "properties"),
+          "Creating WLST properties file failed");
+
+      String localhost = "localhost";//"host.docker.internal";
+      Properties p1 = new Properties();
+      p1.setProperty("admin_username", ADMIN_USERNAME_DEFAULT);
+      p1.setProperty("admin_password", ADMIN_PASSWORD_DEFAULT);
+      p1.setProperty("admin_host", localhost);
+      p1.setProperty("admin_port", forwardedPort);
+      assertDoesNotThrow(() -> p1.store(new FileOutputStream(wlstPropertiesFile), "wlst properties file"),
+          "Failed to write the WLST properties to file");
+      logger.info("WLST property file is: {0} ", wlstPropertiesFile.getAbsolutePath());
+
+      // cp WLST script and prop files to the docker container
+      copyFileToDockerContainer(containerName, wlstScriptFilePath, wlstScriptDestPath);
+      copyFileToDockerContainer(containerName, wlstPropertiesFile.getAbsolutePath(), wlstPropDestPath);
+
+      Path filePath = Path.of(wlstPropertiesFile.getAbsolutePath());
+      String content = Files.readString(filePath, StandardCharsets.US_ASCII);
+      logger.info("Content of WLST property file: {0} ", content);
+
+      // accessing WLS vis WLST using the forwarded port
+      result = executeWLSTScriptInDockerContainer(containerName, wlstScriptDestPath, wlstPropDestPath);
+    } catch (Exception ex) {
+      logger.info("Failed to access WLS vis WLST using the forwarded port!");
+      ex.printStackTrace();
+    } finally {
+      stopWlsDockerContainer(containerName);
+      removeWlsDockerContainer(containerName);
+    }
+
+    return result;
+  }
+
+  /**
+   * Create a WebLogic docker container.
+   *
+   * @param domainUid domain uid
```

Comment:
Why we need domainUid parameter to start a docker Container ?

Reply:
to  start the WLS using "/u01/oracle/user_projects/domains/domainUid/startWebLogic.sh"